exercises/practice/leap/tests.ua

```diff
@@ -0,0 +1,11 @@
+~ "leap.ua" ~ Leap
+
+⍤⤙≍ 0 Leap 2015 # year not divisible by 4 in common year
```

Comment:
Don't we want the test comment above the line? These will become *extremely* long otherwise

Reply:
Agreed. I put them at the end of the line to "fix" something, but I must have done something wrong.
Fixing this in https://github.com/exercism/uiua/pull/12/files